TKGame/Animations/ScreenTransistion.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.VisualBasic.FileIO;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StbImageSharp;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Data;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Do you use ALL of these?

Reply:
im wondering if we should at some point go through all the files and just clean up all the `using` statements. ik mine tend to have a good handful too

Reply:
Yes.